lib/wiki_course_edits.rb

```diff
@@ -65,7 +65,8 @@ def enroll_in_course(enrolling_user:)
     @wiki_editor.add_to_page_top(user_page, @current_user, template, summary)
 
     # Add a template to the user's talk page
-    talk_template = "{{#{template_name(@templates, 'user_talk')}|course = [[#{@course.wiki_title}]]"\
+    talk_template = "{{#{template_name(@templates, 'user_talk')}"\
+                    " | course = [[#{@course.wiki_title}]]"\
```

Comment:
You've introduced some behavior changes here with extra spaces, which caused test failures.

In this case, I think that's okay; using spaces between the wiki template parameters is good, and the current behavior does that inconsistently. So I think the best think here will be to tweak the tests for this class to expect the new spacing.